src/logger.py

```diff
@@ -60,10 +60,13 @@ def wrapper(*args, **kwargs):
             # Log exit point, including response summary if possible
             try:
                 if log_enabled:
-                    response_summary = response.get_data(as_text=True)
-                    logger.debug(f"{request.path} {request.method} - Response: {response_summary}")
+                    if hasattr(response, 'get_data'):  # Check if response object has 'get_data' method
```

Comment:
yes i faced an issue while using it , there is an error because response did not have get_data and exception was raised. because of that the other process did not work properly.

Reply:
If that is the case then changes in Python looks good to me.